corehq/apps/export/models/new.py

```diff
@@ -710,9 +704,10 @@ def _merge_schemas(cls, *schemas):
         def resolvefn(group_schema1, group_schema2):
 
             def keyfn(export_item):
-                return'{}:{}'.format(
+                return'{}:{}:{}'.format(
```

Comment:
this was actually a pretty bad bug. we didn't use the export item's transform to define uniqueness (was impossible to do before moving transform to Item)